docs/view_components/energy_system_network.rst

```diff
@@ -33,84 +35,82 @@ Attributes
 
     Properties:
-    - id `error_log`
-
-
+        * id `error_log`
 
 
 Actions
 ^^^^^^^
 
-1. Clicking on the `add es_bus`/`add es_component` button adds a bus or an energy system component to the `es_bus_list`/`es_component_list`, respectively.
-2. Clicking on the `remove_es_bus`/`remove_es_component` button removes the currently selected bus or energy system component from the `es_bus_list`/`es_component_list`, respectively.
-3. Clicking on the `remove_es_bus`/`remove_es_component` button when no bus or energy system component is currently selected sends a log message to the `error_log`.
+1. Clicking on the `add es_bus`/`add es_component` button adds a *Bus* or an *energy system component* to the `es_bus_list`/`es_component_list`, respectively.
+2. Clicking on the `remove_es_bus`/`remove_es_component` button removes the currently selected *Bus* or *energy system component* from the `es_bus_list`/`es_component_list`, respectively.
+3. Clicking on the `remove_es_bus`/`remove_es_component` button when no *Bus* or *energy system component* is currently selected sends a log message to the `error_log`.
 4. Selecting a bus or an energy system component in `es_bus_list`, `es_component_list` or in `network_schema` allows the user to visualise and/or edit its properties in another view-component.
-5. When a the user add a bus or an energy system component by clicking on the `add_es_bus`/`add_es_component`, they can visualise and/or edit its properties in another view-component.
-6. When a the user add a bus or an energy system component by clicking on the `add_es_bus`/`add_es_component`, they can to see it in the `network_schema`.
+5. When a the user add a *Bus* or *energy system component* by clicking on the `add_es_bus`/`add_es_component`, they can visualise and/or edit its properties in another view-component.
+6. When a the user add a *Bus* or *energy system component* by clicking on the `add_es_bus`/`add_es_component`, they can to see it in the `network_schema`.
 
 Requirements
 ^^^^^^^^^^^^
 
-1. each energy bus needs to be connected to at least one energy system component of type source
+1. each energy *Bus* needs to be connected to at least one energy system component of type source
+.. this requirment is not true, see the pull request #79
+
 2. if the user does not provide an energy system component of type sink for an energy bus, the latter is created automatically
-3. notifications informing the user about potential problems with their energy system should be displayed in the `error_log` text area. Problems could be such as a failure to meet any of the other requirements, an undefined property value of an energy system component, or a bus connected to no energy system component
+.. i guess this should be done to dump excess energy flows. The problem is that you can't generalize here. Also, you would probably need a source as slack, in case the sum of energy flows do not fulfill the demand. How about, we phrase it as such: 
```

Comment:
Comment from @srhbrnds : "i guess this should be done to dump excess energy flows. The problem is that you can't generalize here. Also, you would probably need a source as slack, in case the sum of energy flows do not fulfill the demand. How about, we phrase it as such"